src/nile/core/account.py

```diff
@@ -85,26 +85,16 @@ def deploy(self, status_type=None):
 
         return address, index
 
-    def send(
+    def declare(
```

Comment:
wat there were two `send`?

Reply:
Git thinks so! Part of the merging fun :)